packages/cli/src/lib/setup/setupMultihost.ts

```diff
@@ -99,12 +99,12 @@ export class Multihost {
             let warningShown = false;
             if (dbTools.isLocalObjectsDbServer(config.objects.type, config.objects.host, true)) {
                 console.log('Changing objects server to accept connections on all IP addresses.');
-                config.objects.host = '0.0.0.0';
+                config.objects.host = tools.getListenAllAddress();
                 changed = true;
             } else if (config.objects.type === 'redis') {
                 warningShown = true;
                 console.log(
-                    'Please check the binding of redis service. By default it is only local: http://download.redis.io/redis-stable/redis.conf\nChange "bind 127.0.0.1" to "bind 0.0.0.0" or to others.'
+                    `Please check the binding of redis service. By default it is only local: http://download.redis.io/redis-stable/redis.conf\nChange "bind ${tools.getLocalAddress()}" to "bind ${tools.getListenAllAddress()}" or to others.`
```

Comment:
I think redis default is IPv4, so it should be always 127.0.0.1and not "specific" ... so the first change should be reverted

Reply:
I made the assumption, that if someone wants to use ipv6 he has already configured the redis server accordingly, however we can change back

Reply:
This is a good question ... idea was to give guidance for "default Redis configs" ... If someone touched that setting already he should also remember that